src/main/java/org/apache/commons/collections4/bloomfilter/IndexFilter.java

```diff
@@ -72,7 +72,10 @@ public boolean test(final int number) {
         if (number >= size) {
             throw new IndexOutOfBoundsException(String.format("number too large %d >= %d", number, size));
         }
-        return !tracker.test(number) || consumer.test(number);
+        if (tracker.test(number)) {
```

Comment:
You reverted the IndexFilter file but left this change. I think the statements are identical. Since the plan is to replace this with a separate change to the IndexFilter then can you revert this too. The IndexFilter should be unchanged by this PR.